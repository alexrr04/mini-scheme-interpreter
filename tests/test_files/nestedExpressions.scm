(define (square x) (* x x))

(define (main)
    (define x (read))
    (display x)
    (newline)
    (display (square (+ 2 3))) ; 25
    (newline)
    (display (if (null? '(1 2 3))
                "Empty"
                (cons (square 2) x))) ; (4 4 9)
    (newline)
)
