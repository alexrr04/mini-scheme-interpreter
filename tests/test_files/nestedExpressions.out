(4 9)
25
(4 4 9)
(2 3 4)
